testsuite/src/main/java/io/netty/testsuite/transport/socket/SocketFileRegionTest.java

```diff
@@ -229,4 +244,77 @@ public void exceptionCaught(ChannelHandlerContext ctx,
             }
         }
     }
+
+    private static final class FileRegionWrapper implements FileRegion {
+        private final FileRegion region;
+
+        FileRegionWrapper(FileRegion region) {
+            this.region = region;
+        }
+
+        @Override
+        public int refCnt() {
+            return region.refCnt();
+        }
+
+        @Override
+        public long position() {
+            return region.position();
+        }
+
+        @Override
+        @Deprecated
+        public long transfered() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the missing @deprecated Javadoc tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)